Command/InstallCommand.php

```diff
@@ -0,0 +1,246 @@
+     */
+    private function saveUserToDatabase($userName, $userEmail, $userPassword)
+    {
+        $userManager = $this->getContainer()->get('fos_user.user_manager');
+
+        $user = $userManager->createUser();
+        $user->setUsername($userName);
+        $user->setEmail($userEmail);
+        $user->setPlainPassword($userPassword);
+        $user->setEnabled(true);
+
+        $userManager->updateUser($user);
+    }
+
+    private function createUser()
+    {
+        list($userNameOptionValue, $userEmailOptionValue, $userPasswordOptionValue) = $this->getUserOptions();
+
+        $userNameQuestion = new Question(
+            $this->getQuestion('Please enter the user name', $userNameOptionValue),
+            $userNameOptionValue
+        );
+        $userNameQuestion->setValidator([$this->getValidator(), 'validateUserName']);
+
+        $userEmailQuestion = new Question(
+            $this->getQuestion('Please enter the user email', $userEmailOptionValue),
+            $userEmailOptionValue
+        );
+        $userEmailQuestion->setValidator([$this->getValidator(), 'validateUserEmail']);
+
+        $userPasswordQuestion = new Question(
+            $this->getQuestion('Please enter the user password', $userPasswordOptionValue),
+            $userPasswordOptionValue
+        );
+        $userPasswordQuestion->setValidator([$this->getValidator(), 'validateUserPassword']);
+
+        $userName = $this->questionHelper->ask($this->input, $this->output, $userNameQuestion);
+        $userEmail = $this->questionHelper->ask($this->input, $this->output, $userEmailQuestion);
+        $userPassword = $this->questionHelper->ask($this->input, $this->output, $userPasswordQuestion);
+
+        $this->saveUserToDatabase($userName, $userEmail, $userPassword);
+    }
+
+    private function loadFixtures()
+    {
+        $command = $this->getApplication()->find('doctrine:fixtures:load');
+        $input = new ArrayInput([]);
+        $input->setInteractive($this->input->isInteractive());
+
+        $command->run($input, $this->output);
+    }
+
+    private function printIntro()
+    {
+        $this->output->writeln([
+            '',
+            '<bg=blue;fg=white>Welcome to Grossum Installation!</>',
+            ''
+        ]);
+    }
+
+    private function printOutro()
+    {
+        $this->output->writeln([
+            '',
+            '<bg=blue;fg=white>Grossum Installation has been finished</>',
+            ''
+        ]);
+    }
+
+    private function getQuestion($question, $default, $sep = ':')
+    {
+        if (is_bool($default)) {
+            $default = $default ? 'yes' : 'no';
+        }
+
+        return $default ?
+            sprintf('<info>%s</info> [<comment>%s</comment>]%s ', $question, $default, $sep) :
+            sprintf('<info>%s</info>%s ', $question, $sep);
+    }
+
+    /**
+     * @return Validators
+     */
+    private function getValidator()
+    {
+        if (is_null($this->validators)) {
```

Comment:
plese change "is_null($this->validators)" to "null === $this->validators" because second expression faster than first